addons/website/views/website_templates.xml

```diff
@@ -1178,12 +1178,16 @@ Sitemap: <t t-esc="url_root"/>sitemap.xml
 </template>
 
 <template id="website_search_box" name="Website Searchbox">
-    <div class="input-group">
-        <input type="text" name="search" class="search-query form-control oe_search_box" placeholder="Search..." t-att-value="search"/>
-        <div class="input-group-append">
-            <button type="submit" class="btn btn-primary oe_search_button" aria-label="Search" title="Search"><i class="fa fa-search"/></button>
+    <form class="o_wsale_products_search_bar_input o_website_sale_search" t-att-action="action if action else '/shop'" method="get">
```

Comment:
You have to think about everything when write code, not just your task...

1. You add '/shop' in website... can't be good
2. You add a <form/> in website but I see this template is called x times by many apps and all define their own form, so you will have a form in a form and break those x apps.
3. ...

When you remove a t-call from, check it was not inherited to do stuff, when you change a template check where it was t-called to see if it does not break the feature at that location, etc.